bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -116,7 +124,19 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
 
         me.trigger('indicator.changed', regionsets.length > 0);
     },
-  
+    datasourceSelected: function (datasrc, regionsets) {
```

Comment:
Why is this needed? Isn't parameters affected by indicatorselection, not datasource selection?